core/math/geometry_2d.h

```diff
@@ -350,6 +350,8 @@ class Geometry2D {
 		return triangles;
 	}
 
+	// Assumes cartesian coordinate system (e.g. +x to the right, +y up).
+	// If using screen coordinates (+y down) the result will need to be flipped.
```

Comment:
The description in the docs is more correct IMO. The screen coordinate system is also Cartesian AFAIK, i.e. defined by two orthogonal axes that cross at the origin.

From Wikipedia:

> In mathematics, physics, and engineering, the first axis is **usually** defined or depicted as horizontal and oriented to the right, and the second axis is vertical and oriented upwards. **(However, in some [computer graphics](https://en.wikipedia.org/wiki/Computer_graphics) contexts, the ordinate axis may be oriented downwards.)** The origin is often labeled O, and the two coordinates are often denoted by the letters X and Y, or x and y. The axes may then be referred to as the X-axis and Y-axis. The choices of letters come from the original convention, which is to use the latter part of the alphabet to indicate unknown values. The first part of the alphabet was used to designate known values.

(Emphases mine.)

Well long comment to say it shouldn't say "e.g. +x to the right, +y up" - it's not an example, it's the requirement.

Reply:
Yeah I was admittedly a bit more careful for the docs version as more people will read it. :grin: